src/components/app/details/triggerView/CIMaterialModal.tsx

```diff
@@ -41,7 +41,7 @@ export const CIMaterialModal = ({
 
     return (
         <VisibleModal className="" close={closeCIModal}>
-            <div className="modal-body--ci-material h-100 flexbox-col" onClick={stopPropagation}>
+            <div className="modal-body--ci-material h-100 flexbox-col w-100" onClick={stopPropagation}>
```

Comment:
This fixes the dancing width of ci material on commit message length or sidebar tab change 🎉